### PR TITLE
Minor: disable process_components for some flags

### PR DESCRIPTION
**Pull request by JimFuller-RedHat**
do not need to resolve process_components with --search-latest… and --search-provides

### CODE REVIEW

File: griffon/services/core_queries.py [hunk 1]
```diff
@@ -602,7 +588,6 @@ def execute(self, status=None) -> List[Dict[str, Any]]:
                 search_all_upstreams_params["type"] = self.component_type
             if not (self.include_inactive_product_streams):
                 search_all_upstreams_params["active_streams"] = "True"
-            search_all_upstreams_params["released_components"] = "True"
```
Comment (JakubFrejlach) [inline griffon/services/core_queries.py:605]: Note: Why we don't want to filter on released components? Broader search?

File: griffon/services/core_queries.py [hunk 2]
```diff
@@ -241,8 +241,8 @@ def async_retrieve_upstreams(self, purl):
 
 def process_component(session, c):
     """perform any neccessary sub retrievals."""
-    # we only need to resolve on REDHAT namespace components (for now)
-    if c.namespace == "REDHAT":
+    # only process provided components
+    if c.software_build is None:
```
Comment (JakubFrejlach) [inline griffon/services/core_queries.py:245]: Why do we restrict this only on components with no software builds?

**Review by JakubFrejlach — COMMENTED** [review]
I have some general questions before approve.

**Review by JakubFrejlach — APPROVED** [review]

File: griffon/services/core_queries.py (hunk 1 above)
Comment (JimFuller-RedHat) [reply griffon/services/core_queries.py:605]: we need broader search for upstreams

File: griffon/services/core_queries.py (hunk 2 above)
Comment (JimFuller-RedHat) [reply griffon/services/core_queries.py:245]: existence of software builds is another way of determining if a component is root or not ... if it is not a root then we should process - if it is a root, there is no need to

**Pull request merged**